components/style/values/specified/font.rs

```diff
@@ -12,9 +12,11 @@ use parser::{Parse, ParserContext};
 use properties::longhands::system_font::SystemFont;
 use std::fmt;
 use style_traits::{ToCss, StyleParseErrorKind, ParseError};
-use values::computed::{font as computed, Context, NonNegativeLength, ToComputedValue};
+use values::computed::{font as computed, Context, Length, NonNegativeLength, ToComputedValue};
 use values::specified::{LengthOrPercentage, NoCalcLength};
-use values::specified::length::FontBaseSize;
+use values::specified::length::{AU_PER_PT, AU_PER_PX, FontBaseSize};
+
+const DEFAULT_SCRIPT_MIN_SIZE_PT: ::std::os::raw::c_uint = 8;
```

Comment:
It might be best to get this from bindings like the previous approach. We can wrap the new structs and its impls with `#[cfg(feature = "gecko")]` since it's only used in gecko.

Reply:
I suggested doing this, because we're removing the Gecko constant anyway, and the less build-time dependent stuff we have the better. It should probably be a `u32` though.

Reply:
Thanks emilio for clarification. I'll update the type to `u32`.